src/components/header/header.js

```diff
@@ -0,0 +1,298 @@
+                    </li>
+
+                    <li class="social-icon">
+                        <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M9.52396 6.77561L15.481 0H14.0695L8.89474 5.88196L4.76466 0H0L6.24687 8.89533L0 16H1.41143L6.87281 9.78701L11.2353 16H16L9.52396 6.77561ZM7.59017 8.97342L6.95627 8.08709L1.92041 1.04119H4.08866L8.15394 6.72961L8.78518 7.61594L14.0689 15.0096H11.9006L7.59017 8.97342Z" fill="#0E0F0C"/>
+                        </svg>  
+                    </li>
+                </div>
+
+                <div class="mobile-menu">
+                    <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <rect width="32" height="32" rx="12" fill="#C3DDBA"/>
+                        <path d="M15.9959 22H16.0049" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M21.9998 22H22.0088" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M9.99981 22H10.0088" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M15.9959 16H16.0049" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M15.9998 10H16.0088" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M21.9998 16H22.0088" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M21.9998 10H22.0088" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M9.99981 16H10.0088" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        <path d="M9.99981 10H10.0088" stroke="#0E0F0C" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+                    </svg>
+                </div>
+
+                <ul class="mobile-menu-items">
+                        <li class="nav-bar-menu-item">Docs</li>
+                        <li class="nav-bar-menu-item">Guides</li>
+                        <li class="nav-bar-menu-item">Blog</li>
+                </ul>
+                 
+            </div>
+
+
+        </header>
+
+
+        <!-- Mobile menu Overlay -->
+        <div class="overlay"></div>
+
+        <!-- Close button -->
+        <button class="close-button">×</button>
+
+
+        <style>
+
+            * {
+                margin: 0;
+                padding: 0;
+                -webkit-font-smoothing: antialiased;
+                -moz-osx-font-smoothing: grayscale;
+                box-sizing: border-box;
+            }
+        
+            header {
+                height: 5.5rem;
```

Comment:
are we not able to leverage more open props values here instead of hardcoding our own odd values?

Reply:
Maybe i missed it, but the issue seems to be that open-props doesn't have values for 5.5, 11rem etc. and i can't seem to figure out how to customize with these values we have in the design.

Reply:
I will try and take a look, but i think generally most design systems work off of even values generally?  What is closest to what we need vs what is open props?